pkgs/development/python-modules/python-neutronclient/default.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
```

Comment:
By the way, do we really need the python-neutronclient package in NixOS (and the dependencies on os-client-config and osprofiler packages)?

Because since the OpenStack Zed release, python-neutronclient has been deprecated and the code has been migrated to [OpenStack Client](https://opendev.org/openstack/python-openstackclient) (OCS).

See

* https://lists.openstack.org/archives/list/openstack-discuss@lists.openstack.org/message/KTTBW2DNGELWHKOXMMBAQYTKKQDX2VNF/
* https://docs.openstack.org/python-neutronclient/latest/contributor/transition_to_osc.html

Reply:
I have followed the [requirements.txt](https://opendev.org/openstack/python-octaviaclient/src/branch/master/requirements.txt#L11) of octaviaclient. 

I do not have checked octaviaclient but it may use some old bindings performed by the neutron client

I will check this day

Reply:
neutronclient is still used for the bindings in octaviaclient.  
There are calls in order to list networks, subnets, ports and qos policies on load balancers
https://opendev.org/openstack/python-octaviaclient/src/branch/master/octaviaclient/osc/v2/utils.py#L192

Reply:
Oops sorry, indeed octaviaclient depends on neutronclient package.